### ISSUE TITLE
[Settings] : Screen Reader is not announcing as 'Saved' and 'Changes Discarded', when user activates 'Save' and 'Discard changes' button.

### ISSUE THREAD

**Issue by Saiteja341** [microsoft/terminal#11967]
### Windows Terminal version

1.12.3472.0

### Windows build number

22509.1011

### Other Software

**Test Environment: **
OS: Windows 11 Version Dev (OS Build 22509.1011)
App: Windows Terminal Preview
Screen Reader : Narrator

### Steps to reproduce

**Repro Steps: **

1.Open Windows Terminal.
2.Open Settings page using 'Ctr+,'
3.Select any 'list-item' from left pane.
4.Now Navigate in the 'Right pane' via tab key & make any changes.
5.Now activate 'Save' or 'Discard changes' button.
6.Observe the issue.

**User Experience: **
Users who rely on screen reader will get impacted here & may not know whether the changes are discarded or saved successfully on activating respective buttons, if Screen Reader do not announces 'Saved' and 'Changes Discarded' when user activate 'Save' and 'Discard changes' button.

Guideline reference
EN 301 549 V3.2.1: 11.5.2.15 Change notification

Attachment : 
[Screen Reader is not announcing as 'Saved' and 'Changes Discarded'.zip](https://github.com/microsoft/terminal/files/7727877/Screen.Reader.is.not.announcing.as.Saved.and.Changes.Discarded.zip)

### Expected Behavior

Screen Reader should narrate as 'Saved' and 'Changes Discarded' as a confirmation message, when user activates 'Save' and 'Discard changes' button.

### Actual Behavior

Screen Reader is not announcing as 'Saved' and 'Changes Discarded', when user activates 'Save' and 'Discard changes' button.
